ambari-metrics/ambari-metrics-assembly/pom.xml

```diff
@@ -837,13 +894,43 @@
                 <data>
                   <src>${collector.dir}/target/embedded/${hbase.folder}</src>
                   <type>directory</type>
-                  <excludes>bin/**,bin/*,lib/*tests.jar</excludes>
+                  <excludes>bin/**,bin/*,lib/*tests.jar,lib/hadoop*jar,lib/guava-11.0.2.jar,lib/commons-beanutils-core-1.8.0.jar</excludes>
```

Comment:
Can we add line break after the comma for better readability ?

Reply:
I tried it and unfortunately it doesn't work. The vafer jdeb [documentation](https://github.com/tcurdt/jdeb/blob/master/docs/maven.md)  also includes the following:

`includes/exludes: 	A comma seperated list of files to include from the directory or tarball`